### PR TITLE
Facility locator vet centers and wait times

### PR DESCRIPTION
**Pull request by bshyong**
Adds vet centers and wait times for Facility Locator
also Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3259

Most of the changes have already been reviewed in previous PRs; this PR combines all of them and adds a content change to resolve https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3259

### CODE REVIEW

**Comment by patrickvinograd** [comment]
@bshyong This looks like it needs some adjustment based on the vets-api PR I put up: https://github.com/department-of-veterans-affairs/vets-api/pull/1054

There were some changes to the wait time categories and API attribute names, and a minor change to the effective date for satisfaction.

**Comment by bshyong** [comment]
Will also add in design and content changes from this issue https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3261#issuecomment-308214734

File: test/e2e/facility-helpers.js [hunk 1]
```diff
@@ -113,23 +112,23 @@ const resultsData = {
         },
         feedback: {
           health: {
-            effective_date_range: 'Jun 2016 - Nov 2016',
-            primary_care_routine: 0.73,
-            primary_care_urgent: 0.74,
-            specialty_care_routine: null,
-            specialty_care_urgent: null
+            effectiveDateRange: 'Sep 2016 - Feb 2017',
```
Comment (patrickvinograd) [inline test/e2e/facility-helpers.js]: This test data needs updating based on new API format.

File: test/e2e/facility-helpers.js [hunk 2]
```diff
@@ -113,23 +112,23 @@ const resultsData = {
...
           health: {
-            primary_care_wait_days: 5.977272,
-            primary_care_wait_sample_size: 44,
-            specialty_care_wait_days: 4.327272,
-            specialty_care_wait_sample_size: 110,
-            mental_health_wait_days: 3.962962,
-            mental_health_wait_sample_size: 27,
-            urgent_consult_percentage: null,
-            urgent_consult_sample_size: null
+            primaryCareWaitDays: 5.977272,
```
Comment (patrickvinograd) [inline test/e2e/facility-helpers.js]: This test data needs updating based on new API format - removal of Days from key, change to individual specialty care categories.

**Review by patrickvinograd — APPROVED** [review]
Minor comments about test data needing update. Approving to enable testing.

**Pull request merged**